### ISSUE TITLE
Feature: allow user to supply customer managed user-identity ID

### ISSUE THREAD

**Issue by JeffGiroux** [F5Networks/terraform-azure-bigip-module#34]
User would like to use an existing user-identity for role based permissions. This helps teams that do not have access to create user-identities...but simply are allowed to use them instead.

## Current Behavior
The current module creates a resource for user-identity and then applies that user-identity to the VM instance resource.

```
  identity {
    type         = "UserAssigned"
    identity_ids = [azurerm_user_assigned_identity.user_identity.id]
  }
```

## Expected Behavior
User should be able to use an existing user-identity by supply an input parameter value

```
variable "user_identity" {
  type        = string
  default     = null
  description = "The ID of the managed user identity to assign to the BIG-IP instance"
}
```

**Comment by KrithikaChidambaram** [comment]
Thanks for reporting. This is being tracked internally: INFRAANO-855

**Comment by JeffGiroux** [comment]
This seems fixed...or in the code at least. Please confirm. I was scanning for this change and see it. However, I do not see it closed as an issue nor mentioned in release notes.

**Comment by KrithikaChidambaram** [comment]
It will be in the release this week.

**Comment by RavinderReddyF5** [comment]
Hi @JeffGiroux , Issue fixed in v1.2.5
